addon/serializers/json-api.js

```diff
@@ -571,8 +571,9 @@ const JSONAPISerializer = JSONSerializer.extend({
   */
   serializeHasMany(snapshot, json, relationship) {
     var key = relationship.key;
+    var shouldSerializeHasMany = isEnabled('ds-check-should-serialize-relationships') ? 'shouldSerializeHasMany' : '__shouldSerializeHasMany';
```

Comment:
Unfortunately this doesn't work so the feature is correctly stripped in production. You need to change this to something like:

```js
var shouldSerializeHasMany = '__shouldSerializeHasMany';
if (isEnabled("ds-check-should-serialize-relationships")) {
  shouldSerializeHasMany = 'shouldSerializeHasMany';
}
```